UI configuration for kubeOVN docs (#1099)
harvester/docs: docs/networking/kubeovn-vm-isolation.md

For more information about the schema and usage guidelines, see [Subnet ACL](https://kubeovn.github.io/docs/v1.13.x/en/guide/subnet/#subnet-acl) and [ACL API Reference](https://kubeovn.github.io/docs/v1.13.x/en/reference/kube-ovn-api/#acl) in the Kube-OVN documentation.

### Configuration

See [Subnet Settings](./kubeovn-vpc.md#subnet-settings) for basic subnet configuration.

On the **Access Control List** tab, click Add Rule and configure the following options:

- Action – Specify the action to apply to traffic that matches the rule.
- Direction – Define whether the rule applies to Ingress or Egress traffic.
- Priority – Set the rule priority. Rules with higher priority are evaluated before rules with lower priority.

> - Match – Define the traffic criteria for the rule, such as matching the source or destination IP address.

Reviewer: ```suggestion
See [Subnet Settings](./kubeovn-vpc.md#subnet-settings) for information about basic subnet configuration.

On the **Access Control List** tab, define traffic rules with the following components:

- **Action**: Action to take when traffic matches the rule.
- **Direction**: Traffic flow to which the rule applies (ingress or egress).
- **Priority**: Integer that determines when the rule is evaluated. Rules with larger values are evaluated first (for example, `1000` takes precedence over `100`).
- **Match**: Packet filtering criteria, such as the source or destination IP address.
```

Author: Modified as "Direction of traffic flow to which the rule applies (ingress or egress)"

:::

### Configuration

1. On the Harvester UI, go to **Overlay Networks > Policies**.

1. Click **Create**.

1. Configure the following settings under **Ingress Rules** or **Egress Rules** tab.

   1. Click `Add Allowed traffic source` to add one or more rules.

   1. Click `AddAllowed Port`	to specify the port number and protocol.

   ![](/img/np-ingress.png)

>    ![](/img/np-egress.png)

Reviewer: ```suggestion
1. On the **Ingress Rules** tab, configure the following settings:

    - **Sources**: IP blocks, namespaces, or pods from which inbound traffic is permitted.
    - **Allowed ports**: Network ports and protocols open to inbound traffic.

    ![](/img/np-ingress.png)

1. On the **Egress Rules** tab, configure the following settings:

    **Targets**: IP blocks, namespaces, or pods to which outbound traffic is permitted.
    **Allowed ports**: Network ports and protocols permitted for outbound traffic.

    ![](/img/np-egress.png)
```

1. Configure **Selectors**

   These selectors define the pods to which the NetworkPolicy is applied.

   ![](/img/np-selectors.png)

1. Configure **Labels and Annotations**

   - Label - Key-value pairs attached to the NetworkPolicy resource for identification, organization, and filtering.

   - Annotations - Key-value pairs used to store additional metadata, such as ownership or tool-specific information.

>    ![](/img/np-labels.png)

Reviewer: ```suggestion
1. On the **Selectors** tab, define rules that select the workload pods to which this network policy applies.

   ![](/img/np-selectors.png)

1. On the **Labels and Annotations** tab, configure the following settings:

    - **Labels**: Key-value pairs used to categorize and filter the `NetworkPolicy` resource.
    - **Annotations**: Non-identifying metadata used by external tools or system controllers.

    ![](/img/np-labels.png)
```
